modules/react/loading-sparkles/README.md

```diff
@@ -0,0 +1,5 @@
+# Canvas Kit Loading Sparkles
+
+View the
+[documentation for Loading Sparkles](https://workday.github.io/canvas-kit/?path=/docs/components-indicators-loading-sparkles-react--basic)
+on Storybook.
```

Comment:
I'm not sure if we want to add any additional information that indicates this is intended for only AI-related features.